examples/.testcaferc.json

```diff
@@ -63,6 +63,7 @@
         "rejectUnauthorized": true
     },
     "tsConfigPath": "/Users/s.johnson/testcafe/tsconfig.json",
+    "disablePageCaching": true,
```

Comment:
I think example should be "copy-paste ready", probably it'd better to set "safe" values?

Reply:
We would have to remove half of these options to make it copy-paste ready. Do we need it?